dashboard/test/ui/runner.rb

```diff
@@ -299,11 +299,12 @@ def output_synopsis(output_text)
       link = "https://test-studio.code.org/ui_test/" + html_output_filename
       message += " <a href='#{link}'>☁ html output</a>"
     end
+    short_message = message
 
     message += "<br/><i>command line: cucumber #{arguments + first_time_arguments}</i>"
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers message, color: 'red' if CDO.hip_chat_logging
+    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
```

Comment:
what shorter thing? as far as I can tell it is the same thing (short_message = message)